# config/vlp16_lexus_cluster.param.yaml
---

/**:
  ros__parameters:
    use_cluster: True
    use_box: True
    max_cloud_size: 55000
    downsample: False
    use_lfit: True
    use_z: True
    cluster:
      frame_id: "base_link"
      min_cluster_size: 10
      max_num_clusters: 256
    hash:
      min_x: -130.0
      max_x:  130.0
      min_y: -130.0
      max_y:  130.0
      side_length: 0.7
    voxel:
      min_point:
        x: -130.0
        y: -130.0
        z: -130.0
      max_point:
        x: 130.0
        y: 130.0
        z: 130.0
      voxel_size:
        x: 0.2
        y: 0.2
        z: 0.2
